integration-tests/tests/run-analysis.js

```diff
@@ -0,0 +1,62 @@
+// This test is owned by the Interactive Analysis (IA) Team.
+const _ = require('lodash/fp')
+const { withRegisteredUser, withBilling, withWorkspace, performAnalysisTabSetup, createAnalysis } = require('../utils/integration-helpers')
+const {
+  click, clickable, findElement, noSpinnersAfter, fillIn, findIframe, findText, dismissNotifications
+} = require('../utils/integration-utils')
+
+
+const notebookName = 'test-notebook'
+
+const testRunAnalysisFn = _.flow(
+  withWorkspace,
+  withBilling,
+  withRegisteredUser
+)(async ({ workspaceName, page, testUrl, token }) => {
+  await performAnalysisTabSetup(page, token, testUrl, workspaceName)
+
+  // Create analysis file
+  await createAnalysis(page, notebookName)
+  // Close the create cloud env modal that pops up
+  await noSpinnersAfter(page, {
+    action: () => findText(page, 'A cloud environment consists of application configuration, cloud compute and persistent disk(s).')
+  })
```

Comment:
Yes, because there are actually two spinners that happen after create analysis. One happens when the notebook 'create' is clicked and the bucket call refreshes the background page, which is waited for in the `createAnalysis` function, and one happens when the cloud env creation modal loads, which is what this one waits for.